packages/components/src/FilterDateRangePicker/subcomponents/FilterDateRangePickerField/utils/getDateValidationHandler.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { LabelledMessage } from "~components/LabelledMessage"
+import { getNodeText } from "../../../../utils/getNodeText"
+import { ValidationMessage, DateValidationResponse } from "../types"
+
+export type GetDateValidationHandlerArgs = {
+  onValidate: ((validationResponse: DateValidationResponse) => void) | undefined
+  setInbuiltValidationMessage: (
+    validationMessage: ValidationMessage | undefined
+  ) => void
+  inputLabel: React.ReactNode
+}
+
+export const getDateValidationHandler =
```

Comment:
Could this be `dateValidationHandler`? Doesn't seem to return a function that would be considered a handler of any kind.

Reply:
This is a curry function. We are returning a function 😄 Do you want me to make a type for the return value instead of typing the inner function's args and return value?

Reply:
Ohh I totally missed the double `=>` nah it's fine.